torch/onnx/symbolic.py

```diff
@@ -674,6 +674,10 @@ def topk(g, self, k, dim=None, largest=True, sorted=True, out=None):
     return g.op("TopK", self, k_i=k, axis_i=dim, outputs=2)
 
 
+def repeat(g, self, repeats):
+    return g.op("Tile", self, g.op("Constant", value_t=torch.LongTensor(repeats)))
```

Comment:
We should check `len(repeats) == len(self.type().sizes())`.
If not:
- `len(repeats) < len(self.type().sizes())`, then add `1`s to the second operand.
- `len(repeats) > len(self.type().sizes())`, probably call `Reshape` + `Tile`

Reply:
@houseroad 
Let me try. I am not familiar with pytorch.
Does it accept repeats not be the same as the number of dimensions in input?

Reply:
@houseroad 
I get `RuntimeError: Number of dimensions of repeat dims can not be smaller than number of dimensions of tensor`
when `len(repeats) < len(self.type().sizes())`. So I think we could ignore this case.

I will work on reverse one.